sdk/tests/conformance/extensions/angle-instanced-arrays.html

```diff
@@ -277,6 +280,17 @@
     wtu.checkCanvasRect(gl, Math.ceil(0.25*canvas.width), 0, w, h, [0, 0, 255, 255]);
     wtu.checkCanvasRect(gl, Math.ceil(0.75*canvas.width), 0, w, h, [255, 255, 0, 255]);
 
+    debug("");
+    debug("Testing drawArraysInstancedANGLE with attributes 'divisor' reset to 0");
+    ext.vertexAttribDivisorANGLE(colorLoc, 0);
```

Comment:
Could you please add another debug() call saying what the correct output is, and what the incorrect rendering looked like when this driver bug was occurring?